xbmc/Util.cpp

```diff
@@ -1575,9 +1575,9 @@ void CUtil::InitRandomSeed()
 }
 
 #ifdef TARGET_POSIX
-bool CUtil::RunCommandLine(const std::string& cmdLine, bool waitExit)
+bool CUtil::RunCommandLine(const std::vector<std::string> argArray, bool waitExit)
 {
-  std::vector<std::string> args = StringUtils::Split(cmdLine, ",");
+  std::vector<std::string> args = argArray;
```

Comment:
There is no need to copy the vector two times. As it is passed by value you can directly operate on it without changing anything for the calling function.

Reply:
no problems.